tests/Resources/surveys/test_get_surveys_list.py

```diff
@@ -50,6 +52,21 @@ def test_get_surveys_list_todo(self, mock_request):
         self.assertTrue('collection_exercise'.encode() in response.data)
         self.assertTrue('business_party'.encode() in response.data)
 
+    @requests_mock.mock()
+    def test_get_surveys_list_todo_before_go_live(self, mock_request):
+        mock_request.get(url_get_case_by_party, json=case)
+        mock_request.get(url_get_collection_exercise, json=collection_exercise_before_go_live)
+        mock_request.get(url_get_business_party, json=business_party)
+        mock_request.get(url_get_survey, json=survey)
+        mock_request.get(url_get_collection_instrument_size, text="5\n")
+
+        response = self.app.get(self.test_todo_url, headers=self.headers)
+
+        self.assertEqual(response.status_code, 200)
+        self.assertFalse('case'.encode() in response.data)
+        self.assertFalse('collection_exercise'.encode() in response.data)
+        self.assertFalse('business_party'.encode() in response.data)
```

Comment:
(This might require changing a lot of the unit test files - something for another PR?)

Does the endpoint return JSON? If so, can we load the response and check the contents that way rather than looking for individual words in the raw data? I think doing so would increase readability of what we're actually testing for.

Reply:
That definitely seems like a better way to do things. We've started doing it in a few other repos. Might be good to do it across the whole repo in it's own PR

Reply:
Tech debt [card created](https://trello.com/c/vV2BRjiF).